bpmn-model/src/main/java/io/camunda/zeebe/model/bpmn/validation/zeebe/FlowElementValidator.java

```diff
@@ -89,9 +89,9 @@ public void validate(
         && !NON_EXECUTABLE_ELEMENT_TYPES.contains(elementType)) {
       validationResultCollector.addError(
           0,
-          "An un-supported element found. The XML is valid, but "
-              + "currently, just a sub-set of BPMN elements are supported, see the documentation for "
-              + "more details: https://docs.camunda.io/docs/reference/bpmn-processes/bpmn-coverage");
+          "Elements of type 'elementType.getSimpleName()' are currently not supported. Please refer "
+              + "to the documentation for a list of supported elements: "
+              + "https://docs.camunda.io/docs/reference/bpmn-processes/bpmn-coverage");
```

Comment:
❌ Maybe I should have clarified myself better 😄 We don't want to actually return `elementType.getSimpleName()` in the message. We want to be executed.

```suggestion
          String.format(
              "Elements of type '%s' are currently not supported. Please refer "
                  + "to the documentation for a list of supported elements: "
                  + "https://docs.camunda.io/docs/reference/bpmn-processes/bpmn-coverage",
              elementType.getSimpleName()));
```